client/gutenberg/extensions/contact-info/phone/save.js

```diff
@@ -11,7 +11,11 @@ export function renderPhone( inputText ) {
 			if ( number.trim() === '' ) {
 				return number;
 			}
-			const just_number = number.replace( /\D/g, '' );
+			let just_number = number.replace( /\D/g, '' );
```

Comment:
A super duper nit but we should use camelCase for `justNumber`, because consistency. :-)

Reply:
```suggestion
			let justNumber = number.replace( /\D/g, '' );
```